### PR TITLE
Feature/achievements

### PR DESCRIPTION
**Pull request by Ishadijcks**
Achievements are pretty much done.

When we merge new features we can make some more achievements for them.

https://www.reddit.com/r/PokeClicker/comments/6pnk9i/dear_pok%C3%A9clickers_we_need_your_help_with/

https://docs.google.com/spreadsheets/d/1oO5eocClR-LAtegKK8oRlIMBxRDiq9v-rs4Iu4z0nho/edit#gid=0

### CODE REVIEW

**Comment by Nyargleblargle** [comment]
The "collect 100 (whatever)balls" achievements display as 10/10.

**Comment by Ishadijcks** [comment]
Fixed thanks

File: src/scripts/GameConstants.ts [hunk 1]
```diff
@@ -55,6 +55,15 @@ namespace GameConstants {
     export const DUNGEON_CHEST_SHOW = 2;
     export const DUNGEON_MAP_SHOW = 4;
 
+    // Achievements
+    export const ACHIEVEMENT_TICK = 1000;
```
Comment (Aegyo) [inline src/scripts/GameConstants.ts]: This should probably be with the rest of the game ticks

Comment (Ishadijcks) [inline src/scripts/GameConstants.ts]: Moved it

**Pull request merged**